ledger/metrics/src/main/scala/com/daml/metrics/ExecutionMetrics.scala

```diff
@@ -14,9 +14,11 @@ import com.daml.metrics.api.MetricHandle.{
 }
 import com.daml.metrics.api.{MetricDoc, MetricName}
 
+import scala.annotation.nowarn
+
 class ExecutionMetrics(
     prefix: MetricName,
-    factory: MetricsFactory,
+    @nowarn @deprecated factory: MetricsFactory,
```

Comment:
and in the next step, we remove the factory and just used `labeledMetricsFactory`, right?

Reply:
Yes, but leaving up to the owner of the metrics the migration of the existing metrics from the old factory to the new `labeledMetricsFactory`